vscode-trace-extension/src/test/extension-test.spec.ts

```diff
@@ -16,8 +16,8 @@ test('Open Trace from Explorer', async ({ page }) => {
 
 test('Open Trace from Trace Viewer', async ({ page }) => {
     await page.getByRole('tab', { name: 'Trace Viewer' }).locator('a').click();
-    await page.getByRole('button', { name: 'Opened Traces Section' }).hover();
-    await page.getByRole('button', { name: 'Open Trace' }).click();
+    await page.getByRole('button', { name: 'Welcome' }).hover();
+    await page.getByRole('button', { name: 'Open Folder' }).click();
```

Comment:
I think `Open Trace` is till correct and should not be changed.

Reply:
addressed